client/src/components/Q&A/Helpful.jsx

```diff
@@ -1,16 +1,21 @@
-import React, { useState } from 'react';
-
-export default function Helpful() {
-  const [count, setCount] = useState(Math.floor(Math.random() * 5));
+import React, { useContext } from 'react';
+import ProductContext from '../../context';
+// need the question helpfulness
 
+export default function Helpful({ helpfulness }) {
+  const { setShowAddAnswerModal } = useContext(ProductContext);
   function handleCount() {
-    setCount(() => count + 1);
   }
 
   return (
     <span
       className="helpful"
-    >Helpful? <u onClick={handleCount}>Yes</u>({count}) | <u>Add Answer</u>
+    >Helpful?
+      <u
+        onClick={handleCount}
+      > Yes
+      </u>
+      ({helpfulness}) | <span onClick={() => setShowAddAnswerModal(true)}><u>Add Answer</u></span>
```

Comment:
maybe mush 14 - 17 unto the same line, this is hard for me to read, personally.

Reply:
agreed!